packages/types/src/runner-config.ts

```diff
@@ -2,16 +2,29 @@
 
 import { RunnerContainerConfiguration } from "./sth-configuration";
 
-export type RunnerConfig = {
+type CommonSequenceConfig = {
+    id: string;
+    sequencePath: string;
     name: string;
-    container: RunnerContainerConfiguration;
     version: string;
+}
+
+export type DockerSequenceConfig = CommonSequenceConfig & {
```

Comment:
This is adapter specific, should be beside the adapter.

Reply:
not sure what you mean

Reply:
Meaning this shouldn't be in types, but there should be a place for this beside the adapter. Let's leave this for now, but that'll need revisiting when we address Kube.